onnxruntime/contrib_ops/cpu/qlinear_lookup_table.cc

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "qlinear_lookup_table.h"
+#include <limits>
+#include <cmath>
+
+namespace onnxruntime {
+namespace contrib {
+
+void QLinearLookupTableTransform(const uint8_t* x, const uint8_t* table, uint8_t* y, size_t n) {
+  for (; n >= 4; n -= 4) {
+    const size_t x_value0 = x[0];
+    const size_t x_value1 = x[1];
+    const size_t x_value2 = x[2];
+    const size_t x_value3 = x[3];
+    x += 4;
+    const uint8_t table_value0 = table[x_value0];
+    const uint8_t table_value1 = table[x_value1];
+    const uint8_t table_value2 = table[x_value2];
+    const uint8_t table_value3 = table[x_value3];
+
+    y[0] = table_value0;
+    y[1] = table_value1;
+    y[2] = table_value2;
+    y[3] = table_value3;
+    y += 4;
+  }
+  for (; n != 0; --n) {
+    const size_t x_value0 = *x++;
+    const uint8_t table_value0 = table[x_value0];
+    *y++ = table_value0;
+  }
+}
+
+template <typename T>
+Status QLinearLeakyRelu<T>::Compute(OpKernelContext* context) const {
+  const float X_scale = *(context->Input<Tensor>(1)->Data<float>());
+  const T X_zero_point = (nullptr == context->Input<Tensor>(2)) ? static_cast<T>(0) : *(context->Input<Tensor>(2)->template Data<T>());
+  const float Y_scale = *(context->Input<Tensor>(3)->Data<float>());
+  const T Y_zero_point = (nullptr == context->Input<Tensor>(4)) ? static_cast<T>(0) : *(context->Input<Tensor>(4)->template Data<T>());
+
+  constexpr int qmin = std::numeric_limits<T>::min();
+  constexpr int qmax = std::numeric_limits<T>::max();
+  std::vector<T> table(256);
+  for (int i = 0; i < 256; ++i) {
+    T x = static_cast<T>(i);
```

Comment:
>static_cast<T>(i); [](start = 10, length = 18)

How about: for(int i = numeric_limits<T>::min(); i<=numeric_limits<T>::max(); i++)

Reply:
This is the tricky part. As table treat all as unsigned int. We can not use i + qmin here. since 0 is 0 for int. 128(int)  is -128(int8_t)....